### PR TITLE
Add `__contains__` to `CountMinSketch`

### LINKED ISSUE #32: __contains__ doesn't work with python3
#### Code to Reproduce
```python
from bounter import bounter

lines = [
    ['some', 'sentence'],
    ['another', 'sentence'],
    ['here', 'have', 'some', 'more'],
    ['you', 'should', 'stop', 'now'],
]

counter = bounter(size_mb=1024, need_iteration=False, log_counting=1024)

for line in lines:
    bigrams = zip(line, line[1:])
    counter.update(line)
    counter.update(' '.join(pair) for pair in bigrams)

print(counter['some'])
>>> 2

print('some' in counter)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-16-f89596bc8ae8> in <module>()
----> 1 print('some' in counter)

/home/jayant/miniconda3/envs/pii_tools_env/lib/python3.5/site-packages/bounter/count_min_sketch.py in __getitem__(self, key)
    127 
    128     def __getitem__(self, key):
--> 129         return self.cms.get(key)
    130 
    131     def cardinality(self):

TypeError: The parameter must be a unicode object or bytes buffer!

print(b'some' in counter)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-17-071c2b90c294> in <module>()
----> 1 print(b'some' in counter)

/home/jayant/miniconda3/envs/pii_tools_env/lib/python3.5/site-packages/bounter/count_min_sketch.py in __getitem__(self, key)
    127 
    128     def __getitem__(self, key):
--> 129         return self.cms.get(key)
    130 
    131     def cardinality(self):

TypeError: The parameter must be a unicode object or bytes buffer!

```

Is `__contains__` supported? I haven't tried with python2.

### PR DESCRIPTION
**Pull request by aneesh-joshi**
Fixes https://github.com/RaRe-Technologies/bounter/issues/32

Ran the test provided by @jayantj on python 2 and 3

```python
from bounter import bounter

lines = [
    ['some', 'sentence'],
    ['another', 'sentence'],
    ['here', 'have', 'some', 'more'],
    ['you', 'should', 'stop', 'now'],
]

counter = bounter(size_mb=1024, need_iteration=False, log_counting=1024)

for line in lines:
    bigrams = zip(line, line[1:])
    counter.update(line)
    counter.update(' '.join(pair) for pair in bigrams)

print(counter['some'])
>>> 2

print('some' in counter)
>>> True

print('mikolov' in counter)
>>> False
```

### CODE REVIEW

**Comment by aneesh-joshi** [comment]
ping @menshikh-iv

File: bounter/count_min_sketch.py [hunk 1]
```diff
@@ -128,6 +128,9 @@ def table_size(width, depth=4, log_counting=None):
     def __getitem__(self, key):
         return self.cms.get(key)
 
+    def __contains__(self, item):
+        return self.cms.get(item)
```
Comment (jayantj) [inline bounter/count_min_sketch.py]: Does this return a boolean?

Comment (aneesh-joshi) [reply bounter/count_min_sketch.py]: yes

It runs like described in the comment above: 
https://github.com/RaRe-Technologies/bounter/pull/38#issue-225933667

**Comment by menshikh-iv** [comment]
@aneesh-joshi thank you :1st_place_medal:

**Pull request merged**